builds/e2e/connectivity.yaml

```diff
@@ -111,9 +120,9 @@ jobs:
         displayName: 'Download Edgelet Artifacts'
         inputs:
           buildType: specific
-          project: $(vsts.project)
-          pipeline: $(edgelet.package.build)
-          branchName: $(edgelet.package.branchName)
+          project: $(resources.pipeline.packages.projectID)
+          pipeline: $(resources.pipeline.packages.pipelineName)
+          branchName: $(resources.pipeline.packages.sourceBranch)
```

Comment:
`$(edgelet.package.branchName)` is being used in two other places in this file...can be replaced with `$(resources.pipeline.packages.sourceBranch)`

Reply:
Fixed.